src/test/java/com/matthewglover/tictactoe/core/MiniMaxTest.java

```diff
@@ -23,7 +26,7 @@ public void selectsDrawingMoveOverLosingMove() {
         // 7 o 9
         Game game = new Game(3);
```

Comment:
Just noticed this parameter to Game's constructor. Is this the board size? Should the game have to know about that detail?